scripts/release/mule/sign/sign.sh

```diff
@@ -22,14 +22,16 @@ OS_TYPE=$(./scripts/release/mule/common/ostype.sh)
 ARCHS=(amd64 arm arm64)
 ARCH_BITS=(x86_64 armv7l aarch64)
 
-# It seems that copying/mounting the gpg dir from another machine can result in insecure
-# access privileges, so set the correct permissions to avoid the following warning:
-#
-#   gpg: WARNING: unsafe permissions on homedir '/root/.gnupg'
-#
-find /root/.gnupg -type d -exec chmod 700 {} \;
-find /root/.gnupg -type f -exec chmod 600 {} \;
-
+if ./scripts/release/mule/common/running_in_docker.sh
+then
+    # It seems that copying/mounting the gpg dir from another machine can result in insecure
+    # access privileges, so set the correct permissions to avoid the following warning:
+    #
+    #   gpg: WARNING: unsafe permissions on homedir '/root/.gnupg'
+    #
+    find /root/.gnupg -type d -exec chmod 700 {} \;
```

Comment:
Maybe we should use an env var to set this path, then we could just always package that in our docker images. This may not always work since not every container runs as root. As we develop our automation we'll want to move away from root containers since there are some vulnerabilities with them

Reply:
I'll change this by adding another env var to the package-sign.yaml file.  It will be optional and will default to /root if not set.

I think optional is better than forcing that to be defined, since in the overwhelming number of use cases it will be /root.  You bring up a good point though, and once we move away from root containers this can be revisited (and hopefully this will since have been rewritten in python).